common/scripted_triggers/wc_dislike_units_triggers.txt

```diff
@@ -0,0 +1,63 @@
+dislike_aerial_trigger = {
+	OR = {
+		culture_group = culture_group:hozen_group
+		culture_group = culture_group:kobold_group
+		culture_group = culture_group:murloc_group
+		culture_group = culture_group:trogg_group 
+		culture_group = culture_group:gnoll_group
+		culture_group = culture_group:magnataur_group
+		culture_group = culture_group:tuskarr_group
+		culture_group = culture_group:centaur_group
+		culture_group = culture_group:quilboar_group
+		culture_group = culture_group:furbolg_group
+		culture_group = culture_group:pygmy_group
+		culture = culture:blood_troll
+		culture = culture:yaungol
+		culture = culture:drust
+		culture = culture:giblin
+		culture = culture:saurok
+	}
+}
+
+dislike_massive_aerial_trigger = {
+	OR = {
+		culture_group = culture_group:hozen_group
+		culture_group = culture_group:kobold_group
+		culture_group = culture_group:murloc_group
+		culture_group = culture_group:trogg_group 
+		culture_group = culture_group:gnoll_group
+		culture_group = culture_group:magnataur_group
+		culture_group = culture_group:tuskarr_group
+		culture_group = culture_group:centaur_group
+		culture_group = culture_group:quilboar_group
+		culture_group = culture_group:furbolg_group
+		culture_group = culture_group:pygmy_group
+		culture_group = culture_group:harpy_group
+		culture = culture:blood_troll
+		culture = culture:yaungol
+		culture = culture:drust
+		culture = culture:giblin
+		culture = culture:saurok	
+	}
+}
+
+dislike_siege_weapons_trigger = {
+	OR = {
+		culture_group = culture_group:hozen_group
+		culture_group = culture_group:kobold_group
+		culture_group = culture_group:murloc_group
+		culture_group = culture_group:trogg_group 
+		culture_group = culture_group:gnoll_group
+		culture_group = culture_group:magnataur_group
+		culture_group = culture_group:tuskarr_group
+		culture_group = culture_group:centaur_group
+		culture_group = culture_group:quilboar_group
+		culture_group = culture_group:furbolg_group
+		culture_group = culture_group:pygmy_group
+		culture = culture:blood_troll
+		culture = culture:yaungol
+		culture = culture:drust
+		culture = culture:giblin
+		culture = culture:saurok
+	}
+}
```

Comment:
This is cancer that going to kill the AI potential of many cultures. Siege weapons are the MOST needed unit.